### DISCUSSION TITLE
Python: full course for beginners

### DISCUSSION THREAD

**Discussion by mrspecht** [community/community#113119] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Show & Tell

### Body

[Python for beginners](https://github.com/mrspecht/python-essentials)

**Comment by Annam679** [comment]
I am currently learning the fundamentals. I will make sure to check it since everything looks very neat and organized on first glance. Thank you!

**Reply by mrspecht** [reply]
Nice. My plan is to focus on examples, instead of boring descriptions. Don't forget to leave a star if you like the content. Thanks

**Comment by queenofcorgis** [comment]
Hi @mrspecht , we’re glad you’re here 😀

Thanks for sharing your project with the community. Unfortunately, we currently do not allow self-promotion or advertising on the Community Discussions. We want to make sure there is space for users to ask questions without overwhelming them with other conversations. Thank you for helping us maintain a productive and tidy community for all our members.
